lib/ssh_scan/result.rb

```diff
@@ -142,12 +142,8 @@ def set_end_time
     end
 
     def scan_duration
-      if start_time.nil?
-        raise "Cannot calculate scan duration without start_time set"
-      end
```

Comment:
I thought having exception handling here was going to be helpful, but all it did was make it harder to spec out the results object, so I'm removing it.